archive/d/dart/prime-number.dart

```diff
@@ -17,20 +17,20 @@ void main(List<String> args){
       return;
     }
 
-    // If number is even other than 2; number is multiple of 5 other than 5; if it is 0; it is 1
-    if( (num_needed == 0)|| (num_needed == 1) || (num_needed %2 ==0 && num)needed != 2) || (num_needed %2 ==0 && num)needed != 2) ){
+    // If the number is even number other than 2 OR if the number is multiple of 5 other than 5 or if number is either 0 or 1, Print Composite
+    if( (num_needed == 0)|| (num_needed == 1) || (num_needed %2 ==0 && num_needed != 2) || (num_needed %5  ==0 && num_needed != 5) ){
```

Comment:
The check for 5 is not needed. If 5 is given the `for` loop will not run. If a multiple of 5 is given, the `for` loop will catch that.